test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -305,6 +312,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "Missing X-Message-Type header"
```

Comment:
The generated error message could be changed to the following:<br>`A 'X-Message-Type' header must be defined in the request.`